src/KioskMode/View.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/* Copyright (c) 2018 - Richard Klees <richard.klees@concepts-and-training.de> - Extended GPL, see LICENSE */
+
+namespace ILIAS\KioskMode;
+
+use ILIAS\UI;
+
+/**
+ * A kiosk mode view on a certain object. See README/Architecture for further
+ * details and README/Implementing a Provider for further directions about
+ * implementation.
+ */
+interface View {
+	/**
+	 * Build an initial state based on the Provided empty state.
+	 */
+	public function buildInitialState(State $empty_state) : State;
```

Comment:
How is this done? Could you give an example?

Reply:
Yes. Most simple example could just be:

```
public function buildInitialState(State $empty_state) : State {
    return $empty_state;
}
```

If you'd like to start at the first page of something, the implementation could look like this:

```
public function buildInitialState(State $empty_state) : State {
    return $empty_state->withValueFor("current_page", "1");
}
```